tests/jsunit/variable_model_test.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * @license
+ * Blockly Tests
```

Comment:
The Travis tests are failing because the uncompressed files are being used and I'm not pushing that. All tests pass locally on my machine. Code Review onward!!